src/app/external-log-in/email-confirmation/confirm-email/confirm-email.component.html

```diff
@@ -0,0 +1,37 @@
+<h4 class="mb-3">
+  {{ "external-login.confirm-email.header" | translate }}
+</h4>
+
+<form [formGroup]="emailForm" (ngSubmit)="submitForm()">
+  <div class="form-group">
+    <input
```

Comment:
For accessibility, all inputs should have a `label`. This appears to be missing a label.